src/components/organisems/Header.js

```diff
@@ -45,6 +53,7 @@ export default function Header({ siteTitle, isFront }) {
           <HeaderLink className={"focus:underline hover:underline"} to="/about">About</HeaderLink>
         </NavBar>
       </HeaderContainer>
+        <ActionBotton/>
```

Comment:
Moving this inside of the `NavBar` component will probably put it where you want it. 

There's also a typo in the component and filename: `ActionBotton` -> `ActionButton`.

Moving the `href` and `text` here: `<ActionButton href="csp.rustfest.eu">Submit Talk</ActionButton>` will allow us to re-use the button in other places.